pkg/operators/wasm/maps.go

```diff
@@ -76,6 +78,35 @@ func (i *wasmOperatorInstance) addMapFuncs(env wazero.HostModuleBuilder) {
 	)
 }
 
+func isMapTypeKnown(typ ebpf.MapType) bool {
+	return typ <= ebpf.TaskStorage
+}
```

Comment:
nit: add comment to keep it aligned with wasmapi/go/map.go? 

I was checking why only ebpf.TaskStorage given that new types maps were added in https://github.com/cilium/ebpf/blob/v0.17.1/types.go, I found out it's because on wasmapi/go/map.go we only have up to TaskStorage.